sandbox/settings.py

```diff
@@ -105,6 +105,8 @@
 
 STATIC_URL = '/static/'
 
-TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'), )
+TEMPLATE_DIRS = (os.path.join(BASE_DIR, 'accounts/templates'),
+os.path.join(BASE_DIR, 'templates')
+)
```

Comment:
a co z places/templates ?
ps. skoro to trzeba dodać to jakim cudem to wrześniej działało ?